apps/www/app/_components/tokens-comp/typography/typography-previews.tsx

```diff
@@ -0,0 +1,22 @@
+export const LineHeight = ({ value }: { value: string }) => {
+  return (
+    <div style={{ lineHeight: value }}>
+      line <br /> height
+    </div>
+  );
+};
+
+export const FontSize = ({ value }: { value: string }) => {
+  return <div style={{ fontSize: value }}>Aa</div>;
+};
+
+export const FontWeight = ({ value }: { value: string; text: string }) => {
+  return <div style={{ fontWeight: value }}>weight</div>;
```

Comment:
```suggestion
  return <div style={{ fontWeight: value }} lang='en'>weight</div>;
```
I think we can keep things like these as english